docs/ocis/development/testing.md

```diff
@@ -539,6 +539,57 @@ BEHAT_FEATURE="tests/acceptance/features/apiAntivirus/antivirus.feature" \
 make test-acceptance-api
 ```
 
+## Running Test Suite With Federated Sharing (@ocm)
+Test suites that are tagged with `@ocm` require running two different ocis instances. More detailed information and configuration related to it can be found [here](https://doc.owncloud.com/ocis/5.0/deployment/services/s-list/ocm.html).
+
+Put
+```bash
+127.0.0.1 ocis-server
+127.0.0.1 federation-ocis-server
+```
+in the `/etc/hosts` file
+
+### Setup first ocis instance
+
+```bash
+# run oCIS
+OCIS_URL="https://ocis-server:9200" \
+PROXY_ENABLE_BASIC_AUTH=true \
+GRAPH_INCLUDE_OCM_SHAREES=true \
+OCM_OCM_INVITE_MANAGER_INSECURE=true \
+OCM_OCM_SHARE_PROVIDER_INSECURE=true \
+OCM_OCM_STORAGE_PROVIDER_INSECURE=true \
+OCM_OCM_PROVIDER_AUTHORIZER_PROVIDERS_FILE="${workspaceFolder}/tests/config/drone/providers.json" \
+OCIS_ADD_RUN_SERVICES="ocm"
+ocis/bin/ocis server
+```
+
+The federation-ocis-server will be available at: https://ocis-server:9200/
+
+### Setup second ocis instance
+#### Using .vscode/launch.json
+
+The federation-ocis-server will be available at: https://federation-ocis-server:10200/
```

Comment:
```suggestion
The federation-ocis-server will be available at: https://ocis-server:9200/

### Setup second ocis instance
#### Using .vscode/launch.json

The federation-ocis-server will be available at: https://federation-ocis-server:10200/
```
Can you rewrite/rephrase **The federation-ocis-server will be available** because we use same on both place.